src/mam4xx/wet_dep.hpp

```diff
@@ -2061,6 +2114,7 @@ class WetDeposition {
 
   const char *name() const { return "MAM4 Wet Deposition"; }
 
+  KOKKOS_INLINE_FUNCTION
```

Comment:
It should be just an `inline` function as it is called from the host, right?

Reply:
Yes, "init" is generally not callable from device since it can allocate memory.